### PR TITLE
Restructure guides to section, update quickstart

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

File: content/vendor/guides/quickstart/_index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 date: "2020-02-20T00:00:00Z"
 lastmod: "2020-02-20T00:00:00Z"
-title: "Quickstart"
-redirect: "/vendor/quickstart/create-release"
+title: "Guides"
+redirect: "/vendor/guides/quickstart/create-release"
```
Comment (marccampbell) [inline content/vendor/guides/quickstart/_index.md:5]: @dexhorthy will the current URL be redirected so we don't get 404s?

Comment (dexhorthy) [reply content/vendor/guides/quickstart/_index.md:5]: good catch, yeah let me update that. thanks.

**Comment by austinchambers** [comment]
Pulling from master so that we can ensure CI runs properly for this branch.

File: content/vendor/guides/quickstart/_index.md (hunk 1 above)
Comment (dexhorthy) [reply content/vendor/guides/quickstart/_index.md]: done

**Comment by austinchambers** [comment]
Not quite working properly. "Existing Cluster" appears twice: 
image

**Comment by dexhorthy** [comment]
@austinchambers pushed another version

**Comment by austinchambers** [comment]
@dexhorthy I like how it's collapsed onto a single page! So much easier to work with now, I think. 

However, it looks like we've got some broken links: 
```
  *  internally linking to ../../../vendor/guides/quickstart/automate, which does not exist (line 0)
htmlproofer 3.15.0 | Error:  HTML-Proofer found 6 failures!
     <a href="../../../vendor/guides/quickstart/automate">CLI setup chapter</a>
  *  internally linking to ../create-release, which does not exist (line 0)
     <a href="../create-release">Create and Promote a Release</a>
  *  internally linking to ../create-release, which does not exist (line 0)
     <a href="../create-release">creating a release</a>
  *  internally linking to ../install, which does not exist (line 0)
     <a href="../install">installing</a>
  *  linking to internal hash # that does not exist (line 0)
     <a href="../../../vendor/guides/quickstart/#">CLI setup guide</a>
```

**Comment by austinchambers** [comment]
Still a bit of weirdness on the Guides section: 
image

**Comment by dexhorthy** [comment]
@austinchambers I don't see this -- can you include the url path in your screenshot? 
Screen Shot 2020-04-07 at 11 07 51 AM

**Comment by austinchambers** [comment]
It's from the deploy preview. Click the details link 
image

**Comment by dexhorthy** [comment]
@austinchambers can you include the URL route still? I can't reproduce, and I don't know what page to check.

Screen Shot 2020-04-07 at 11 12 28 AM

**Comment by austinchambers** [comment]
https://deploy-preview-149--kotsio.netlify.com/vendor/guides/existing-cluster/create-release/

**Comment by austinchambers** [comment]
This is now what I see: https://deploy-preview-149--kotsio.netlify.com/vendor/guides/existing-cluster/
image

**Comment by dexhorthy** [comment]
ah @austinchambers gotcha, thanks now I see it. For some reason `Existing Cluster` doesn't show up in the side nav in my local environment, but appears in the netlify preview. Trying a few more frontmatter flags.

**Comment by dexhorthy** [comment]
okay @austinchambers fixed with some restructuring https://deploy-preview-149--kotsio.netlify.com/vendor/guides/

**Pull request merged**